notebooks/Combine_Libraries.ipynb

```diff
@@ -0,0 +1,3663 @@
+{
```

Comment:
<p>did you mean to delete "containing" and end the sentence?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
<p>Thanks for catching that! Fixed now.</p>